addons/website/views/website_views.xml

```diff
@@ -128,6 +128,48 @@
             </field>
         </record>
 
+        <record id="website_page_properties_view_form" model="ir.ui.view">
+            <field name="name">website.page.properties.form.view</field>
+            <field name="model">website.page</field>
+            <field name="arch" type="xml">
+                <form>
+                    <notebook>
+                        <page string="Name">
+                            <group class="mt-4">
+                                <field name="name" string="Page Name" class="form-control"/>
+                            </group>
+                            <group>
+                                <field name="website_id" groups="website.group_multi_website" invisible="1"/>
+                                <field name="url" string="Page URL" widget="page_url" required="True"/>
+                            </group>
+                        </page>
+                        <page string="Publish">
+                            <group class="mt-4">
+                                <field name="is_in_menu" string="Show in Top Menu" widget="boolean_toggle"/>
+                            </group>
+                            <group>
+                                <field name="is_homepage" string="Use as Homepage" widget="boolean_toggle"/>
+                            </group>
+                            <group>
+                                <field name="website_indexed" string="Indexed" widget="boolean_toggle"/>
+                            </group>
+                            <group>
+                                <field name="is_published" string="Publish" widget="boolean_toggle"/>
```

Comment:
Nitpicking: `Indexed` vs `Publish` -> Should both be in past form with `ed` or both not, don't you think?
I'd go for `Published`, the same as in the navbar (the toggle)